app/objects/c_story.py

```diff
@@ -138,9 +143,28 @@ def _adjust_icon_svgs(path):
             viewbox = [int(float(val)) for val in icon_svg.get('viewBox').split()]
             aspect = viewbox[2] / viewbox[3]
             icon_svg.set('width', str(round(float(icon_svg.get('height')) * aspect)))
-            icon_svg.set('x', '-' + str(int(icon_svg.get('width')) / 2))
+            if not icon_svg.get('id') or 'legend' not in icon_svg.get('id'):
+                icon_svg.set('x', '-' + str(int(icon_svg.get('width')) / 2))
         svg.write(open(path, 'wb'))
 
+    @staticmethod
+    def _get_table_object(val):
+        table = ParagraphStyle(name='Table', fontSize=8)
+        if type(val) == str:
```

Comment:
any reason to not use `isinstance` for these checks, as it's a bit more flexible?

Generally, it's frowned on to check for specific types or use isinstance in Python, but here it looks OK.